resources/assets/js/videos/videoContainer.vue

```diff
@@ -673,6 +673,12 @@ export default {
             }
             Messages.danger(`Invalid shape. ${shape} needs ${count} different points.`);
         },
+        selectLastAnnotation() {
+            let lastAnnotation = this.annotations.reduce((lastAnnotated, a) => a.id > lastAnnotated.id ? a : lastAnnotated, { id: 0 });
+            let remainingAnnotations = this.annotations.filter(a => a.id != lastAnnotation.id);
+            lastAnnotation.selected = true;
+            remainingAnnotations.map(a => a.selected = false);
```

Comment:
Have you tried this and it works? When an annotation is selected (e.g. in the sidebar) the tool should seek the video to the start of the annotation. In the videoContainer `annotation.selected` is then set to this time. Same as above, it should work fine if you reuse `selectAnnotations()`:

```suggestion
            this.selectAnnotations([lastAnnotation], [], lastAnnotation.startFrame);
```

Reply:
Thanks! When I tested it, I created multiple annotations in the same frame, which is why I didn't notice that.

Reply:
Did you leave the second parameter empty on purpose? Because this causes the last annotation only to be selected but not to seek the video, if another annotation was already selected on the timeline.

Reply:
No. I just wanted to avoid `selectAnnotation()` because this can have unexpected side effects with attaching or swapping labels. You should use:

```js
this.selectAnnotations([lastAnnotation], this.selectedAnnotations, lastAnnotation.startFrame);
```